gateway-ha/src/test/java/io/trino/gateway/ha/router/TestTrinoRequestUser.java

```diff
@@ -42,4 +52,32 @@ void testJsonCreator()
         assertThat(deserializedTrinoRequestUser.getUser()).isEqualTo(trinoRequestUser.getUser());
         assertThat(deserializedTrinoRequestUser.getUserInfo()).isEqualTo(trinoRequestUser.getUserInfo());
     }
+
+    @Test
+    void testUserFromJwtToken()
+    {
+        final String claimUserName = "username";
+        final String claimUserValue = "trino";
```

Comment:
Remove final. We don't use it for local variables in Trino project.

Reply:
ok, let me remove it.